aiconfig-docs/docs/gradio-workbook.md

```diff
@@ -0,0 +1,144 @@
+  </picture>
+</div>
+
+#### Local Parameters
+
+You can set local parameters that only apply to the cell you are in. Click on the cell sidebar and **Local Parameters**. Note: Local parameters will override the global parameters if they have the same name.
+
+<div align="center">
+  <picture>
+    <img alt="workbook" src="https://github.com/lastmile-ai/aiconfig/assets/81494782/0935ac7c-5e68-42f1-befe-e279b027cac8" width="800"/>
+  </picture>
+</div>
+
+### Chaining
+
+You can reference the output from one cell in another cell. This allows you to create complex, interconnected workflows within your workbook.
+
+The summarization prompt below uses the output of the story_gen prompt using handlebars syntax
+(e.g `{{story_gen.output}}`)
+
+![chaining](https://github.com/lastmile-ai/aiconfig/assets/81494782/be65e264-3734-422e-9967-35a44d77cbe7)
+
+### Download
+
+Click **Download** to download the `aiconfig` file that captures the current state of your space in a JSON file. Example Space and corresponding `aiconfig` file:
+
+- [Gradio Workbook Space](https://huggingface.co/spaces/lastmileai/ai_story_generator)
+- [my_app.aiconfig.json](https://huggingface.co/spaces/lastmileai/ai_story_generator/blob/main/story_gen.aiconfig.json)
+
+There are 2 purposes of the aiconfig file.
+
+1. **Lock in edits to your space.** Refreshing your space does not save changes. You need to upload the downloaded `aiconfig` file to your space repo to lock in the state. Make sure you name your aiconfig as `my_app.aiconfig.json` - it’s referenced in app.py.
+2. **Build apps inspired by your space.** HuggingFace spaces offer cool demos, but building an app with the tested models and prompts is challenging. However, by downloading the `aiconfig` file, you can use the prompts (and models) from your space in your code with the [AIConfig SDK](https://github.com/lastmile-ai/aiconfig).
+
+### Share Workbook
+
+Click **Share Workbook** to get a link to a read-only version of your space! Even space viewers can showcase your prompts and model outputs with the rest of the world.
+
+![sharing](https://github.com/lastmile-ai/aiconfig/assets/81494782/ceadc825-9df3-4192-b033-117ee1d40590)
+
+### Build apps inspired by Spaces
+
+You can easily build generative apps inspired by your work in Gradio Workbook Spaces! Both space creators and space viewers can download the aiconfig file to capture the state of their space.
+
+Use the aiconfig file in your code with the [AIConfig SDK](<(https://github.com/lastmile-ai/aiconfig)>).
+
+[AIConfig](<(https://github.com/lastmile-ai/aiconfig)>) is an open-source framework, message us on [Discord](https://discord.com/invite/xBhNKTetGx) if you feedback or questions.
+
+## Supported Models
+
+- **Supports models on [Hugging Face Inference API](https://huggingface.co/docs/api-inference/index).** [Hugging Face Tasks](https://huggingface.co/tasks) supported:
```

Comment:
"Gradio Notebooks support models which use the [Hugging Face Inference API](https://huggingface.co/docs/api-inference/index).